preflight_check:

  topic_check:

    topics: [
      "estimation_manager/uav_state:mrs_msgs/msg/UavState",
      "control_manager/diagnostics:mrs_msgs/msg/ControlManagerDiagnostics",
      "some_topic_that_does_not_exist:nav_msgs/msg/Odometry",
    ]
